draft-ietf-dprive-dnsoquic.md

```diff
@@ -590,9 +590,12 @@ Implementation might choose not to use a QUIC API for padding if it is
 significantly simpler to re-use existing DNS message padding logic which is
 applied to other encrypted transports.
 
-In the absence of a standard policy for padding sizes, implementations should
-consider following the recommendations of the Experimental status "Padding
-Policies for Extension Mechanisms for DNS (EDNS(0))" {{?RFC8467}}.
+In the absence of a standard policy for padding sizes, implementations SHOULD
+following the recommendations of the Experimental status "Padding Policies for
+Extension Mechanisms for DNS (EDNS(0))" {{!RFC8467}}. Whilst Experimental,
```

Comment:
i don't want to get into a territorial dispute 😛, but in en_US, "whilst" has been considered archaic for over 100 years.  "While" is simpler language, and arguably more universal, because it's valid in both en_US and en_GB.